corehq/apps/userreports/expressions/list_specs.py

```diff
@@ -61,7 +63,7 @@ def __str__(self):
                                                     map=add_tabbed_text(str(self._map_expression)))
 
 
-class ReduceItemsExpressionSpec(JsonObject):
+class ReduceItemsExpressionSpec(NoPropertyTypeCoercionMixIn, JsonObject):
```

Comment:
(disclaimer: I realize I've broken this in the past [inside UCR code](https://github.com/dimagi/commcare-hq/blob/master/corehq/apps/userreports/es/data_source.py#L25))

I think it's convention to have MixIns after the other objects in the inheritance change. Is that possible here?

Reply:
I think it depends whether you want to override functionality or provide a default. I based this [off of this explanation](https://stackoverflow.com/a/10018792/8207) which I think implies that if I want to override the `JsonObject` metaclass that I should put it first?